File: Tests/executors/CXX/core/src/TokenExecutor.cpp
#include "TokenExecutor.h"

void TokenExecutor::doGet(framework::HttpRequest& request, framework::HttpResponse& response)
{
	const framework::JsonObject data = *request.getTokenPayload();

	response.setBody(data["userName"].get<std::string>());
}

DEFINE_EXECUTOR(TokenExecutor)

File: Tests/executors/CSharp/CoreExecutors/TokenExecutor.cs
using Framework;
using System.Text.Json.Nodes;

public class TokenExecutor : StatelessExecutor
{
	public override void DoGet(HttpRequest request, HttpResponse response)
	{
		JsonObject data = request.GetTokenPayload()!;

		response.SetBody(data["userName"]!.GetValue<string>());
	}
}
